telegram/constants.py

```diff
@@ -865,6 +928,34 @@ class UpdateType(StringEnum):
     """:obj:`str`: Updates with :attr:`telegram.Update.chat_join_request`."""
 
 
+class TitleLimit(IntEnum):
```

Comment:
@Poolitzer suggested changing this to `ChatLimit`, which will mean that after merging this into `master` we will make classes of constants pertaining to `Chat` bigger. The change itself (here) is quick, but I guess a vote might be needed

Reply:
https://t.me/c/1494805131/30064

Reply:
BTW, why was this added in this PR and not in #3351?

Reply:
Change done

Reply:
> BTW, why was this added in this PR and not in #3351?

and this? :D

Reply:
Well I didn't reply to that because IIRC @Poolitzer added these constants and I don't remember the reason stated for that. Anyway, once this is merged into `master`, I will merge `master` into the branch for #3351 and will sort it out from there :)

Reply:
ah, I even reviewed #3343 … time flies. Okay, the title length was changed in API 6.3. so all good.